### PR TITLE
Issue 1/task/project create event

### LINKED ISSUE #1: Add project to the create event.
Currently, creating an event enables the instructor to create an event for exams, assignments, and office hours. Add project to the existing events and provide instructor a flexibility to add an event for the project.

### PR DESCRIPTION
**Pull request by chandur626**
This PR contains changes related to issue-1 that is implementing the project create event functionality. The test cases for the functionality are also added to this PR.

@nirajlavani @SriPallaviDamuluri can you please review this and approve it?

### CODE REVIEW

File: src/event_creation.py [hunk 1]
```diff
@@ -82,9 +83,42 @@ def check(m):
...
+        elif button_clicked == 'project':
+            await ctx.send('What is the title of this project?')
+            msg = await BOT.wait_for('message', timeout=60.0, check=check)
+            project_title = msg.content.strip()
+
+            await ctx.send(
+                'Please enter the due date of this project along with the time\n'
+                'Enter in 24-hour format for e.g. an project due at 11:59pm can be inputted '
+                'as 23:59\nEnter in format `MM-DD-YYYY %H:%M`')
+            msg = await BOT.wait_for('message', timeout=60.0, check=check)
```
Comment (SriPallaviDamuluri) [inline src/event_creation.py:97]: Should we not accept the start date?

**Review by HariniBharata — APPROVED** [review]
Approved.

**Review by SriPallaviDamuluri — APPROVED** [review]

**Review by nirajlavani — APPROVED** [review]
It looks good to me.

**Review by 05sandesh — APPROVED** [review]

File: src/event_creation.py (hunk 1 above)
Comment (chandur626) [reply src/event_creation.py:97]: Does not actually add any importance to the event in the case of the project. Therefore, accepting only due date.

**Pull request merged**